website/cue/reference/remap/literals/string.cue

```diff
@@ -21,6 +21,10 @@ remap: literals: string: {
 		#"""
 			"Hello, world! \\u1F30E"
 			"""#,
+		#"""
```

Comment:
👍 to the new example here. Could we also add a note about this under `characteristics` though? I think that would help the behavior be more discoverable.